README.md

```diff
@@ -1 +1,246 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+<!-- - [❓ FAQ (OPTIONAL)](#faq) -->
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portolio] <a name="about-project"></a>
+
+This Portfolio is a personal project which could help others understand how to parse a Figma design to create a UI, how to use Flexbox to place elements in the page, how to build a personal portfolio site and how to use images and backgrounds to enhance the look of the website.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<!-- 
+> Describe the tech stack and include only the relevant sections that apply to your project. -->
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/curriculum-html-css/blob/main/html5.md/">HTML & CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/linters.md">Linter.yml</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Version Control </summary>
+  <ul>
+    <li><a href="https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/git_github_basics.md">Git & Github</a></li>
+  </ul>
+</details> 
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+
+- **[Includes_a_descriptive_readme_file]**
+- **[Has_Client_side_rendering]**
+- **[It_is_a_simple_application]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> A developer can use this project to run tests on various html&css files
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+Example command:
+
+```sh
+ npm install git
+```
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/danutibe07/Hello-Microverse-Project.git
+```
+
+### Install
+
+Open this project with your desired browser
+
+
+Example command:
+
+```sh
+  code .
+```
+<!-- 
+### Usage -->
+
+<!-- To run the project, execute the following command: -->
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+<!-- ### Run tests -->
+
+<!-- To run tests, run the following command: -->
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+<!-- 
+### Deployment -->
+<!-- 
+You can deploy this project using: -->
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
```

Comment:
It's great to have these sections in your project 💯  but in order to make the `README.md` look more professional you should make sure that each section is relevant to the project 📄 the sections Usage and Deployment are mentioning Rails but this is an HTML, CSS and JS project.  Adjusting that will show recruiters or fellow collaborators that you care about documentation. 😉